doc/releases/nautilus.rst

```diff
@@ -320,29 +315,34 @@ Instructions
    and verify that each monitor has both a ``v2:`` and ``v1:`` address
    listed.
 
-#. For each host that has been upgrade, you should update your
-   ``ceph.conf`` file so that it references both the v2 and v1
-   addresses.  Things will still work if only the v1 IP and port are
-   listed, but each CLI instantiation or daemon will need to reconnect
-   after learning the monitors real IPs, slowing things down a bit and
+#. For each host that has been upgraded, you should update your
+   ``ceph.conf`` file so that it either specifies no monitor port (if
```

Comment:
might want to update https://github.com/ceph/ceph/blob/master/doc/cephfs/nfs.rst#configuration-for-libcephfs-clients to be in sync with this suggestion.

Reply:
probably we can also update https://github.com/ceph/ceph/blob/master/doc/rados/configuration/mon-config-ref.rst#minimum-configuration to have an example using the new syntax, for instance
```
mon host =  [v2:192.168.2.8:40995,v1:192.168.2.8:40996] [v2:192.168.2.8:40997,v1:192.168.2.8:40998] [v2:192.168.2.8:40999,v1:192.168.2.8:41000]
```
and link it to `doc/rados/configuration/msgr2.rst` 's `Updating ceph.conf and mon_host` section.

also could link `doc/rados/configuration/network-config-ref.rst` to the same section.